src/test/configuration_snippet.rs

```diff
@@ -24,19 +24,6 @@ impl ConfigurationSection {
     fn get_section<I: Iterator<Item = String>>(
         file: &mut Enumerate<I>,
     ) -> Option<ConfigurationSection> {
-        lazy_static! {
-            static ref CONFIG_NAME_REGEX: regex::Regex =
-                regex::Regex::new(r"^## `([^`]+)`").expect("failed creating configuration pattern");
-            // Configuration values, which will be passed to `from_str`:
-            //
-            // - must be prefixed with `####`
-            // - must be wrapped in backticks
-            // - may by wrapped in double quotes (which will be stripped)
-            static ref CONFIG_VALUE_REGEX: regex::Regex =
-                regex::Regex::new(r#"^#### `"?([^`]+?)"?`"#)
-                    .expect("failed creating configuration value pattern");
-        }
-
```

Comment:
Removing the `lazy_static!` block here is fine, but I'd like to continue referring to these regex's by name instead of defining them inline with the `static_regex!` macro you added. Can you change this to something like this and refer to the static references below:

```rust
        let config_name_regex = static_regex!(r"^## `([^`]+)`");
        // Configuration values, which will be passed to `from_str`:
        //
        // - must be prefixed with `####`
        // - must be wrapped in backticks
        // - may by wrapped in double quotes (which will be stripped)
        let config_value_regex = static_regex!(r#"^#### `"?([^`]+?)"?`"#);
```